Makefile

```diff
@@ -67,7 +67,7 @@ endif
 PHONY: run-server
 run-server: ## run server locally
 ifeq (, $(shell which jq))
-	go run main.go server
+	go run cmd/pbnj/*.go server
```

Comment:
when `go mod` became the default go team changed it so `go run` doesn't need all the `*.go` files in the cli args. For example:
```
[~/go/src/github.com/tinkerbell/pbnj]─[manny@dellnix]> 
go run main.go --help
Usage of /run/user/1000/tmp/go-build039568299/b001/exe/main:
  -listen-addr string
    	IP and port to listen on for HTTP (default ":9090")
  -log-level value
    	Log level, one of FATAL, PANIC, DPANIC, ERROR, WARN, INFO, or DEBUG
```

Reply:
sweet, good to know. thanks!